common/src/main/scala/uk/ac/wellcome/storage/VersionedHybridStore.scala

```diff
@@ -26,8 +26,8 @@ case class HybridRecord(
     with Sourced
     with Versioned
 
-class VersionedHybridStore @Inject()(
-  sourcedObjectStore: SourcedObjectStore,
+class VersionedHybridStore[T <: Sourced] @Inject()(
```

Comment:
Do we need to subclass from Sourced? I don't think it's integral to the VHS, which just needs an "id". Maybe at some point we want to remove this restriction?

Reply:
That's where we're going: https://github.com/wellcometrust/platform/pull/1604